sw/airborne/modules/decawave/decawave_anchorless_communication.c

```diff
@@ -32,7 +32,7 @@
   on-board values with each-other (for purposes such as relative localization, co-ordination, or collision avoidance).
   This module must be used together with the Decawave DWM1000 running the appropriate Serial Communication code, which can be flashed on the Arduino board.
   The arduino library can be found at:
-    https://github.com/StevenH2812/arduino-dw1000/tree/UWB_localization_v1.0
+    https://github.com/StevenH2812/arduino-dw1000/tree/UWB_onboard
```

Comment:
Yes, that is on the to-do list, at which point we can change the link

Reply:
As discussed a pull request will be created for all the fixes and addition to the original repo here https://github.com/thotro   

Note that this is separate from this pull request, this one can, after fixes suggested by @kirkscheper  as reviewer are implemented) Which as of 20180530 they are as understood from @coppolam

Reply:
note that this s only a remark line where one could find the arduino-dw1000 UWB_onboard code, and so far a valid one... so no blocker for a pull since there is no other URL atm, as soon as the pull and merge with trhe originating repo is availabe, feel free to adjust the remark to another url...